java/Makefile

```diff
@@ -0,0 +1,12 @@
+SNAPSHOT_VERSION = $(shell mvn help:evaluate -Dexpression=project.version -q -DforceStdout 2> /dev/null)
```

Comment:
This is only used in `version`. Consider removing it from global scope.

Reply:
Resolved in 5cea28bd4511eb8f11b0fc1fc44811885a5704e7